spec/services/service_provider_updater_spec.rb

```diff
@@ -10,7 +10,8 @@
     [
       {
         id: 'big number',
-        updated_at: '2010-01-01 00:00:00',
+        created_at: '2010-01-01 00:00:00'.to_datetime,
+        updated_at: '2010-01-01 00:00:00'.to_datetime,
```

Comment:
And calling `.to_datetime` after the fact in the test won't work because when the SP is saved in the DB, the String datetime will be converted to a DateTime using whatever random Timezone the zonebie gem has set, but when you call `.to_datetime` on the original String, it won't take into account that timezone. I couldn't figure out how to call `.to_datetime` with the current zonebie Timezone and didn't think it was worth the extra time to investigate.